### ISSUE TITLE
Set Base Node 00

### ISSUE THREAD

**Issue by XaroRSA** [maniacbug/RF24Network#13]
Not possible to set base node 00, so no way to transmit data to one single master.
Prints error on setting "00N" through serial.

**Comment by grigis** [comment]
I updated the RF24Network lib on my computer and noticed alot of changes since i last used it (year ago).
Still have old version witch is working fine with some tweaks, but latest revision seems to have an problem setting 00 node.
At a fast glance over the code i noticed that there seems to be another sensornet sketch for the 00 receiver node.
Am I missing something or do i have to code the receiver sketch for node 00 myself?

**Comment by XaroRSA** [comment]
Yeah I had to redo the code for node 00. This version doesnt work by default and doesnt allow you  to set 00.
Maniabug seems to have disappeared from the planet :O

**Comment by grigis** [comment]
As this is his hobby, i understand that he has other work to. I know from my own experience, how projects get cold for a while. Still I appreciate his work so far.
For now i'm gonna use my old modified version, as there is no time to redo the receiver node and updating 26 nodes.
Xaro, you could put up an pull request here on GitHub with receiver sketch if you did one already.

**Comment by dansku** [comment]
I got stuck here too, 
can anyone post a node 00 sketch version?

Thanks! :+1:

**Comment by bobibob77** [comment]
Hi! Did anyone solve a problem during this time?
I'm stuck too :(
